tools/ci/tc/decision.py

```diff
@@ -0,0 +1,314 @@
+import argparse
+import json
+import logging
+import os
+import re
+import subprocess
+from collections import OrderedDict
+
+import taskcluster
+from six import iteritems, itervalues
+
+from . import taskgraph
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+logging.basicConfig()
+logger = logging.getLogger()
+
+
+def get_triggers(event):
+    # Set some variables that we use to get the commits on the current branch
+    ref_prefix = "refs/heads/"
+    pull_request = "pull_request" in event
+    branch = None
+    if not pull_request and "ref" in event:
+        branch = event["ref"]
+        if branch.startswith(ref_prefix):
+            branch = branch[len(ref_prefix):]
+
+    return pull_request, branch
+
+
+def fetch_event_data(queue):
+    try:
+        task_id = os.environ["TASK_ID"]
+    except KeyError:
+        logger.warning("Missing TASK_ID environment variable")
+        # For example under local testing
+        return None
+
+    task_data = queue.task(task_id)
+
+    event_data = task_data.get("extra", {}).get("github_event")
+    if event_data is not None:
```

Comment:
Just `return event_data` would have exactly the same effect, but is this to make it explicit that `None` is possible?